loki/files/config.yaml

```diff
@@ -65,6 +65,7 @@ memberlist:
     - ruler
 querier:
   query_ingesters_within: 2h
+  query_timeout: ${LOKI_QUERIER_CONFIG_QUERY_TIMEOUT:100s}
```

Comment:
Please name this env var `LOKI_QUERIER_QUERY_TIMEOUT` -> the pattern I've been using is the path to the field in the yaml.

I think leave the default the same as loki default (`1m` https://grafana.com/docs/loki/latest/configuration/#querier_config)